src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    private static int figureCount = 6;
```

Comment:
Make it constant (static final) with proper naming

Reply:
Done